### PR TITLE
Milestone 1: Linters setup for syntax and stylistic validations

### PR DESCRIPTION
**Pull request by ogagaoghene**
#Linter 

#Description
Linting was configured to highlight syntactic and stylistic problems in our file

### CODE REVIEW

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @ogagaoghene, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [ ] Kindly make sure that the linters are set up correctly and pass with green checkmarks ✔️ . You can use [these linters](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by Microverse. (Please keep in mind that the file is called `linters.yml`, not `blank.yml`).
- [ ] Nice work using the readme template provided by Microverse. Please edit all of its sections and provide information to each one of them to better suit your project. (Project name and description should have the project's title and a brief description, Authors section should contain information on your social links, ...etc).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁 
# Hello  @ogagaoghene 😃 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

- [x] As requested by the previous code reviewer, Kindly make sure that the linters are set up correctly and pass with green checkmarks ✔️. You can use [these linters](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by Microverse. (Please keep in mind that the file is called linters.yml, not blank.yml).

- [x] As requested by the previous code reviewer, Please edit all of its sections and provide information to each one of them to better suit your project. (Project name and description should have the project's title and a brief description, Authors section should contain information on your social links, ...etc).


## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ogagaoghene** [comment]
This is my last code review. I have made all of the requested changes by my code reviewers. I have verified that all checks have passed. All of these changes were done in the new branch I created as instructed. Please kindly give me a positive review so I can proceed to the next step.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
 
 ## Built With
 
-- Major languages
+- html
 - Frameworks
-- Technologies used
+- vs code, github, gitbash
 
 ## Live Demo (if available)
```
Comment (jaflih) [inline README.md:11]: [OPTIONAL]  I kindly suggest you to remove this section because you haven't a demo link

File: README.md [hunk 2]
```diff
@@ -18,11 +15,11 @@
...
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+**I used HTML to implement the Hello World Project, implemented the GitHub flow, and adopted best practices in the process of documenting the project.**
 
 
-To get a local copy up and running follow these simple example steps.
+## To request a local copy of my project, please follow these steps
+Request this url: https://github.com/ogagaoghene/hello-world in your web browser. Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side. Select download Zip option from drop down menu. Once the download is complete you will be able to access my project locally.
 
 ### Prerequisites
```
Comment (jaflih) [inline README.md:24]: [OPTIONAL]  I kindly suggest you to remove this section because you don't need it in this project.

File: README.md [hunk 3]
```diff
@@ -1,15 +1,12 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Project Name
```
Comment (jaflih) [inline README.md:1]: [OPTIONAL] Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), as asked in the requirement : 
image

Here is an example of a professional short summary of changes introduced in the pull request.
image

You can use it to improve your summary.
image

**Review by jaflih — APPROVED** [review]
Hi @ogagaoghene , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
Linters working properly ✔️
Commit messages are meaningful ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**